### ISSUE TITLE
Access denied for MS network drive

### ISSUE THREAD

**Issue by koa43556** [fman-community/fman#481]
Hello,
when going to a network drive \\server\user$ mounted to "M:" fails with below error massage. Most likely the error is connected to the automatically created user share from MS windows server in an enterprise IT system as connecting to a "normal" network drive is possible (not "user$" but just a folder name).

Greetings
Andi

```
Command 'GoTo' raised error.

Traceback (most recent call last):
  File "fman\impl\plugins\command_registry.py", line 106, in _execute_command
  File "C:\Users\myuser\AppData\Local\fman\Versions\1.1.8\Plugins\Core\core\commands\__init__.py", line 1009, in __call__
    self.pane.set_path(url)
  File "fman\__init__.py", line 130, in set_path
  File "fman\impl\widgets.py", line 109, in set_location
  File "fman\impl\model\__init__.py", line 46, in set_location
  File "fman\impl\model\__init__.py", line 43, in set_location
  File "fman\impl\model\__init__.py", line 51, in _set_location
  File "fman\impl\plugins\mother_fs.py", line 120, in resolve
  File "C:\Users\myuser\AppData\Local\fman\Versions\1.1.8\Plugins\Core\core\fs\local\__init__.py", line 149, in resolve
    path = Path(path).resolve()
  File "pathlib.py", line 1109, in resolve
  File "pathlib.py", line 186, in resolve
PermissionError: [WinError 5] Access is denied: 'M:\\'
```

**Comment by mherrmann** [comment]
Hi Andi,

sorry for the trouble and thank you for the report. What is this `user$` share? Is that actually how it's written? Does it contain the `$`? Is it `user$` or (eg.) `andi$`?

Thanks,
Michael

**Comment by koa43556** [comment]
Hi Michael,
the command I typically use for connecting to the network share is:
`if not exist M: net use M: \\MyServer\user$\MyUser`
assuming that the server is called MyServer and the user MyUser. user$ is in fact literally written like this. It is also possible to create the link using the "Map Network Drive" button in Windows Explorer. The result would be the same as above command.
I guess that user$ has sth to do with active directory or similar. Actually I thought, the Windows build-in abstraction using Network Drive should abstract issues like that...

Thank you for taking care!
Andi

**Comment by mherrmann** [comment]
Thank you for the clarifications Andi. I just released [fman 1.1.9](https://fman.io/changelog#v1.1.9). I believe it should fix the bug. I'm closing this issue. If the bug still appears for you, please do re-open.
